app.vue

```diff
@@ -1,10 +1,9 @@
 <template lang="pug">
-main.container.mx-auto.p-8(
-  class="lg:p-10"
-)
-  Header
-  Outpainter
-  Footer
+body.bg-gray-50
```

Comment:
The body-tag will be inherently added by the Nuxt.js framework, so it is not needed here.